spec/takeaway_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:menu) { double :menu, dishes: { :spam => 1 } }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)